diameter_human_waist:  0.15
delta_height: -0.20